introduction.md

```diff
@@ -0,0 +1 @@
+Hola, esta es una introducción al dojo de Deadly Vipers
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.